### DISCUSSION TITLE
Rahmen um ROI's ist nicht da, wie eingestellt

### DISCUSSION THREAD

**Discussion by svenschedlbauer** [jomjol/AI-on-the-edge-device#2585] [Q&A]
Hallo, seit geraumer Zeit habe ich das  Problem, dass sich der eingestellte Ramen nicht dort befindet, wie eingestellt. Woran kann das liegen?
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/110197932/10ec160e-6114-4eea-8976-3c7f19e65970)
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/110197932/60ccf41d-ecb6-49e6-8323-644775d8074c)

So wird es in der Übersicht allerdings dargestellt. :-(
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/110197932/200fb704-b253-43f2-a0ac-05d086d9ddbd)

**Comment by caco3** [comment]
Dein Alignment ist nicht gut genug.

Mach die 2 Alignment Marks so weit weg voneinander wie möglich und nicht zu gross.
